routes.js

```diff
@@ -4,14 +4,20 @@ import { CreateUserController } from './src/users/controllers/createUserControll
 import { AuthController } from './src/auth/controllers/authController.js'
 import {createUserValidator} from './src/users/validators/createUserValidator.js'
 import {loginValidator} from './src/auth/validators/loginValidator.js'
+import { authMiddleware } from './src/shared/middlewares/authMiddleware.js';
+
 const routes = express.Router();
 
 const createUser = new CreateUserController();
 const authUserController = new AuthController(); 
 
-routes.get("/", (req, res) => {
+routes.post("/", authMiddleware, (req, res) => {
     const applicationName = process.env.APP_NAME;
-    res.send(`Hi Node and Docker!!! ${applicationName}`);
+    
+    res.json({
+      "message" : `Hi Node and Docker!!! ${applicationName} - ${req.userName}`
+    });
+
```

Comment:
```suggestion
    return res.json({
      "message" : `Hi Node and Docker!!! ${applicationName} - ${req.userName}`
    });
```

it's always good to return the last function of a route. this helps the node to clean up the stack on the event loop.